.circleci/config.yml

```diff
@@ -277,7 +309,9 @@ workflows:
             - build
           filters:
             branches:
-              only: develop
+              only:
+                - develop
+                - mircea/voyager_deployment
```

Comment:
we should remove also this and add `master`

Reply:
Done